README.md

```diff
@@ -735,115 +724,53 @@ The following command can be executed from the CLI in order to retrieve a set of
+If you are using open source DC/OS, follow these instructions to [pass your HTTP API token to the DC/OS endpoint](https://dcos.io/docs/1.8/administration/id-and-access-mgt/auth-api/#passing-your-http-api-token-to-dc-os-endpoints). 
 
-~~~
-$ TODO command for getting an OAuth token.. once docs describing OAuth exist
-{
-  "token": "uSeR_t0k3n"
-}
+Once you have the authentication token, you can store it in an environment variable and reference it in your REST API calls:
 
+```
 $ export AUTH_TOKEN=uSeR_t0k3n
-~~~
+```
 
-This token is then used to authenticate requests to the Kafka Service:
+The `curl` examples in this document assume that an auth token has been stored in an environment variable named `AUTH_TOKEN`.
 
-~~~
-$ curl -H "Authorization: token=$AUTH_TOKEN" "$DCOS_URI/service/kafka/v1/connection"
-GET /service/kafka/v1/connection HTTP/1.1
+## Using the REST API
 
-{
-    "address": [
-        "10.0.0.211:9843",
-        "10.0.0.217:10056",
-        "10.0.0.214:9689"
-    ],
-    "dns": [
-        "broker-0.kafka.mesos:9843",
-        "broker-1.kafka.mesos:10056",
-        "broker-2.kafka.mesos:9689"
-    ],
-    "vip": "broker.kafka.l4lb.thisdcos.directory:9092",
-    "zookeeper": "master.mesos:2181/kafka"
-}
-~~~
--->
+The following `curl` example demonstrates how to retrieve a set of brokers to connect to.
+ 
 
-## Connection Info Response
+    $ curl -H "Authorization: token=$AUTH_TOKEN" "<dcos_url>/service/kafka/v1/connection"
+
+### Connection Info Response
 
 The response, for both the CLI and the REST API is as below.
 
     {
-        "address": [
-            "10.0.0.211:9843",
-            "10.0.0.217:10056",
-            "10.0.0.214:9689"
-        ],
-        "dns": [
-            "broker-0.kafka.mesos:9843",
-            "broker-1.kafka.mesos:10056",
-            "broker-2.kafka.mesos:9689"
-        ],
-        "zookeeper": "master.mesos:2181/kafka"
-    }
+            "address": [
+                "10.0.0.211:9843",
+                "10.0.0.217:10056",
+                "10.0.0.214:9689"
+            ],
+            "dns": [
+                "broker-0.kafka.mesos:9843",
+                "broker-1.kafka.mesos:10056",
+                "broker-2.kafka.mesos:9689"
+            ],
+            "vip": "broker.kafka.l4lb.thisdcos.directory:9092",
+            "zookeeper": "master.mesos:2181/kafka"
+        }
 
 
-This JSON array contains a list of valid brokers that the client can use to connect to the Kafka cluster. For availability reasons, it is best to specify multiple brokers in configuration of the client.
+This JSON array contains a list of valid brokers that the client can use to connect to the Kafka cluster. For availability reasons, it is best to specify multiple brokers in configuration of the client. Use the VIP to address any one of the Kafka brokers in the cluster.
```

Comment:
@sascala:  Maybe linking to DC/OS VIP documentation here would be good.  I was thinking it's a little unclear that VIPs load balance requests, but this might not be the best place to go into all the details.